.github/workflows/CD.yaml

```diff
@@ -114,8 +127,9 @@ jobs:
       - name: check deployment
         run: ./gradlew :dataland-frontend:npm_run_test${ENVIRONMENT} --no-daemon --stacktrace
         env:
+          TEASER_COMPANY_PERM_IDS: ${{ secrets.TEASER_COMPANY_PERM_IDS }}
           ELECTRON_EXTRA_LAUNCH_ARGS: "--ignore-connections-limit=localhost:8090,proxy,localhost,preview-dataland.duckdns.org,dev-dataland.duckdns.org,dataland-local.duckdns.org,dataland-letsencrypt.duckdns.org"
-          CYPRESS_PREPOPULATE_TIMEOUT_S: ${{ github.event.inputs.useInMemoryStorage && 40 || 900 }}
+          CYPRESS_PREPOPULATE_TIMEOUT_S: ${{ github.event.inputs.useInMemoryStorage && 90 || 900 }}
```

Comment:
Brauchen wir hier die 90 wirklich? Wenn ja, dann würde ich den Default-wert in dem setEnv skript auch anpassen (Da wird der aktuell auf 40 gesetzt), sodass wir Konsistenz zwischen lokalen Tests und CI haben.

Reply:
das war schon der Versuch Konsistenz zu schaffen - in .env.development haben wir auch 90s.
Dann passe ich das im setEnv auch noch an.

Reply:
Ok